gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -217,14 +218,20 @@ def run(argv=None):
     if variant_merger:
       variants |= (
           'MergeVariants' >> merge_variants.MergeVariants(variant_merger))
-    _ = (variants |
+    proc_variants = variants | 'ProcessVaraints' >> beam.Map(
+        processed_variant.create_processed_variant, header_fields,
```

Comment:
@arostamianfar when running the pipeline with DataflowRunner, I get a serialization related error at this stage which says:
PicklingError: Can't pickle <type 'function'>: attribute lookup __builtin__.function failed

I am still trying to figure out what causes this and have some theories, but have not got to the root cause/solution yet. If you have had similar issues before, I appreciate your comments.

Reply:
I spent some time on this and I have found _a_ solution. The problem seems to be that Beam uses cPickle as the default serializer and that cannot serialize the new class ProcessedVariant (I think because it has a list of another class SingleAlternateData).

One fix I have found is to specify the coder in coder_registry, i.e., this new line that I just updated:
`beam.coders.registry.register_coder(processed_variant.ProcessedVariant, beam.coders.coders.DillCoder)`

That said, dill is slower than cPickle so this has some performance implications (see the TODO before this line).